.github/workflows/test-installation.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       max-parallel: 48
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        python-version: [3.7.4, 3.7.12, 3.8, 3.9]
+        python-version: [3.7.5, 3.7.12, 3.8, 3.9]
```

Comment:
oh. Why? Doesn't windows or mac support 3.7.4?

Reply:
https://github.com/pytorch/opacus/runs/4327225058?check_suite_focus=true

Reply:
Ubuntu doesn't, and I think it tries all combinations of os/python/pytorch versions